packages/circuit-ui/components/Popover/Popover.tsx

```diff
@@ -194,21 +187,31 @@ export interface PopoverProps {
    */
   fallbackPlacements?: Placement[];
   /**
-   * Function that is called when closing the Popover.
+   * The element that toggles the Popover when clicked.
    */
-  onClose: (event: Event) => void;
+  component: ({
+    onClick,
+    ref,
+    id,
+  }: {
+    onClick: (event: MouseEvent | KeyboardEvent) => void;
+    ref: Ref<HTMLElement>;
+    id: string;
+  }) => JSX.Element;
```

Comment:
This is missing the aria props.


```suggestion
  component: (props: {
    'onClick': (event: MouseEvent | KeyboardEvent) => void;
    'ref': Ref<HTMLElement>;
    'id': string;
    'aria-haspopup': boolean;
    'aria-controls': string;
}) => JSX.Element;
```

Reply:
True! Wondering why TS isn't complaining about it 🤔 @amelako can you add this when you check if all a11y requirements are met for the Popover?